assets/messages.yml

```diff
@@ -87,24 +69,37 @@ variables:
   awaiting_response:
     - project:
         - mc
-        - mcd
         - mcpe
         - mcl
-        - mclg
         - bds
         - realms
         - web
       value: |-
         ~This issue is being temporarily resolved as {color:#d04437}*Awaiting Response*{color}. Once the requested information has been delivered, the report will be reopened automatically.~
 
+  wai_links:
+    - project: bds
+      value: |-
+        [Full Version History|https://minecraft.wiki/w/Bedrock_Dedicated_Server#History] -- [The official Minecraft feedback site|https://feedback.minecraft.net]
```

Comment:
Wondering if we should link to discord here/instead.

The feature request message lists both

Reply:
Same as https://github.com/mojira/helper-messages/pull/249#discussion_r2049294960